great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -64,6 +64,25 @@ def __getattr__(self, name: str) -> DataAssistantRunner:
 
         return data_assistant_runner
 
+    @classmethod
+    def _register_data_assistant(
+        cls,
+        data_assistant: Type[DataAssistant],
+    ) -> None:
+        """
+        This method registers "DataAssistant" subclass for future instantiation and execution of its "run()" method.
+
+        Args:
+            data_assistant: "DataAssistant" class to be registered
+        """
+        # TODO: <Alex>6/21/2022: Only alias is to be registered (leave formal type registration in as commented).</Alex>
+        # data_assistant_type = data_assistant.data_assistant_type
+        # cls._register(data_assistant_type, data_assistant)
+
+        alias: str | None = data_assistant.__alias__
```

Comment:
@TrangPham I had to bring this back in order to avoid the error of `__alias__` not available in the meta class.  Thanks.